marker_uilints/tests/ui/print_cond_expr.stderr

```diff
@@ -1109,86 +1025,33 @@ warning: print test
                        id: ExprId(..),
                        span: SpanId(..),
                    },
-                   expr: Call(
-                       CallExpr {
+                   expr: Path(
```

Comment:
I suppose a bug/feature was fixed here somehow, because before it was a call expression?

Reply:
Yes, before `x?` was *resugared* to `TryTrait::branch(x)?` which was no correct. The removed path now corrects the resugar to `x?`